com.voxdsp.PoryDrive.yaml

```
@@ -0,0 +1,40 @@
+app-id: com.voxdsp.PoryDrive
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: porydrive
+rename-icon: porydrive
+rename-desktop-file: porydrive.desktop
+rename-appdata-file: porydrive.appdata.xml
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --device=dri
+modules:
+  - name: glfw
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+      - -DBUILD_SHARED_LIBS=ON
+      - -DGLFW_BUILD_EXAMPLES=OFF,
+      - -DGLFW_BUILD_TESTS=OFF
+      - -DGLFW_BUILD_DOCS=OFF
+      - -DGLFW_USE_WAYLAND=1
+    sources:
+      - type: git
+        url: https://github.com/glfw/glfw.git
+        commit: 3eaf1255b29fdf5c2895856c7be7d7185ef2b241
+  - name: porydrive
+    buildsystem: simple
+    build-commands:
+      - cc main.c glad_gl.c -I inc -Ofast -L~/glfw/src/ -lglfw3 -lm -o porydrive
```

```suggestion
      - cc main.c glad_gl.c -I inc -Ofast -L/app/lib -lglfw3 -lm -o porydrive
```

This is the thing with not having a build system: you have to do things manually.